docs/content/guide/bootstrap.ngdoc

```diff
@@ -142,6 +142,17 @@ This is the sequence that your code should follow:
   2. Call {@link angular.bootstrap} to {@link compiler compile} the element into an
   executable, bi-directionally bound application.
 
+## Things to keep in mind
+
+There a few things to keep in mind regardless of automatic or manual bootstrapping:
+
+- There is no explicit support for bootstrapping more than one AngularJS applications on a page, and
```

Comment:
OK, so the fact that we don't run any tests for multiple apps per page (and how could we since Protractor doesn't support manual bootstraping :smiley:), doesn't leave as the option to say it is supported. But I find this statement a little too discouraging.

Also, the [`ngApp` docs][1] say:

> To run multiple applications in an HTML document you must manually bootstrap them using angular.bootstrap instead.

Which gives the impression, it **is** possible/supported to run multiple apps on a page.

--
Maybe this could be re-phrased to something more into the direction: _"It might work, but it's untested, there's shared resources and you might run into prolems (especially with more complex apps), so use at your own risk"_

I don't feel too strongly about it - feel free to ignore this.

[1]: https://docs.angularjs.org/api/ng/directive/ngApp